visualize-admin/visualization-tool
# feat: Add time slider to interactive filters

Contributes to #802.

This PR introduces a basic time slider which can be used to interactively filter a chart based on time dimension.

### How to test
1. Go to `/en/browse/dataset/https%3A%2F%2Fenvironment.ld.admin.ch%2Ffoen%2Fubd0104prod%2F5?dataSource=Int&flag__timeslider=true` (notice the `flag__timeslider`, feature is not available by default).
2. Create a column chart.
3. Change X variable to `Parameter`.
4. Go to Publishing step and enable Time Slider.
5. See that a slider appeared below the chart.
6. Use the slider to filter the data.
7. Change X dimension back to Date and see that a time slider disappeared.
